config/jobs/kubernetes/sig-k8s-infra/periodics.yaml

```diff
@@ -27,3 +27,90 @@ periodics:
         requests:
           cpu: 100m
           memory: 512Mi
+
+- interval: 1h
+  name: ci-kubernetes-unit-eks-canary
+  annotations:
+    testgrid-dashboards: sig-k8s-infra-canaries
+    testgrid-tab-name: ci-kubernetes-unit-eks-canary
+    testgrid-days-of-results: '1'
+    testgrid-create-test-group: 'true'
+  decorate: true
+  cluster: eks-prow-build-cluster
+  extra_refs:
+    - org: kubernetes
+      repo: kubernetes
+      base_ref: master
+      path_alias: k8s.io/kubernetes
+  spec:
+    # unit tests have no business requiring root or doing privileged operations
+    securityContext:
+      # NOTE: these are arbitrary non-root values. They don't exist in the
+      # image and don't need to, the unit tests should only write to TMPDIR
+      runAsUser: 2001
+      runAsGroup: 2010
+    containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230222-b5208facd4-master
```

Comment:
something to consider: egress bandwidth :+)

Reply:
Yes. I want thinking about this. we need to have a ECR private repo where we can mirror `gcr.io/k8s-staging-test-infra/kubekins-e2e`. I don't know how to enforce usage of a specific image.

Reply:
Yes. I want thinking about this. we need to have a ECR private repo where we can mirror `gcr.io/k8s-staging-test-infra/kubekins-e2e`. Probably a blocker before we move more jobs.

Reply:
There's also the prow images, though those should be better cached on the node than anything else since they're *all* jobs, but depending on how things go it may be worth a strategy to mirror them.

The same unit tests we use to enforce other presubmit behaviors could enforce that `gcr.io/k8s-staging-test-infra` is substituted with `ecr/...` for jobs on the known EKS clusters. We'll be wanting to look at enforcing other things that way anyhow (e.g. perhaps guaranteedQOS for pods the same as the k8s-infra-prow-builds cluster)

Reply:
For this very limited subset of CI images, I think s/mirror / make the jobs that push them push to two staging locations (GCR/AR, ECR) might be the most straightforward approach for now.

For non-CI images we're generally promoting things to production and registry.k8s.io will handle this for us, even with things like docker-in-docker, or else they're locally built and run in the job anyhow.